### ISSUE TITLE
Update stylelint-config-standard

### ISSUE THREAD

**Issue by racse1** [kraftvaerk/stylelint-config-kraftvaerk#7]
Hi 😃 

I've tried to just bump a version of `stylelint-config-standard` but got tons of errors during the tests. So probably it makes sense to create a separate issue... It's a little bit dependent on #1 and it probably makes sense to move to Jest first and then fix the tests.
